main.js

```diff
@@ -177,12 +218,19 @@ define(function (require, exports, module) {
 
     function onCheckedStateChange() {
         _preferences.set("checked", Boolean(_command.getChecked()));
+        _preferences.set("colors", _preferences.get("colors"));
         updateEditors();
     }
     
     function onActiveEditorChange(e, editor) {
         updateEditors(editor);
     }
+  
+   function updateColors(e, data) {
+     if (data.ids.indexOf("theme") > -1 || data.ids.indexOf("colors") > -1) {
```

Comment:
Is there a cleaner way to do this, or is this about as clean as it'll get? IDK, it just feels a tad hacky.